sql-plugin/src/main/scala/com/nvidia/spark/rapids/cudf_utils/HostConcatResultUtil.scala

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-package ai.rapids.cudf
+package com.nvidia.spark.rapids.cudf_utils
 
-import ai.rapids.cudf.JCudfSerialization.HostConcatResult
```

Comment:
Nit: Not sure why this change was made, as it made this PR significantly larger than it needed to be.

Reply:
mostly for consistency with how JCudfSerialization.SerializedTableHeader is referenced

Reply:
mostly for consistency with how JCudfSerialization.SerializedTableHeader is referenced